feconf.py

```diff
@@ -259,7 +259,7 @@
 CURRENT_COLLECTION_SCHEMA_VERSION = 6
 
 # The current version of story contents dict in the story schema.
-CURRENT_STORY_CONTENTS_SCHEMA_VERSION = 4
+CURRENT_STORY_CONTENTS_SCHEMA_VERSION = 5
```

Comment:
We don't have to increment the value for Story model. 
This is required for migration of schema of StoryContents (which contains StoryNode)

Reply:
updated